baby-liminal-extension/tests/utils/mod.rs

```diff
@@ -0,0 +1,122 @@
+use std::sync::mpsc::Receiver;
+
+use aleph_runtime::Runtime;
+use baby_liminal_extension::{
+    executor::Executor,
+    substrate::{weight_of_store_key, weight_of_verify, Extension},
+    ProvingSystem, VerificationKeyIdentifier,
+};
+use obce::substrate::{
+    frame_support::weights::Weight, pallet_contracts::chain_extension::RetVal,
+    CallableChainExtension, ChainExtensionEnvironment,
+};
+use scale::{Decode, Encode};
+
+mod environment;
+
+pub use environment::{
+    CorruptedMode, MockedEnvironment, Responder, RevertibleWeight, StandardMode, StoreKeyErrorer,
+    StoreKeyOkayer, VerifyErrorer, VerifyOkayer,
+};
+
+pub const STORE_KEY_ID: u16 =
+    <dyn baby_liminal_extension::BabyLiminalExtension as obce::codegen::MethodDescription<
+        2390688905,
```

Comment:
how would one generate it?

Reply:
The `impl` blocks themselves are generated automatically, while the identifier lookup problem is planned to be solved with a separate macro in OBCE, but for now it's advised to use [`cargo-expand`](https://github.com/dtolnay/cargo-expand) to manually get identifiers.

Reply:
okay for now, we hope for the macro then :P